### ISSUE TITLE
Support jmespath in template operation's write property

### ISSUE THREAD

**Issue by lodejard** [Microsoft/Atlas#46]
it would be nice in an operation like this
```
- template: example-template.yaml
  write: example-output.yaml
```
if the write filename could be late-bound to variables (like message property can be)
```
- template: example-template.yaml
  write: ( ['example-', thing.name, '.yaml'] )
```

**Comment by oneturkmen** [comment]
Hi Louis, I want to take this, but will need a little bit of help :) 

As I understand, we want to be able to pass variables as JMES expression, e.g. `thing.name`, in the template operation's `write` property.

Should I add the new class/interfaces/etc., or what file should be modified instead? I.e. I feel like something should be done somewhere [here](https://github.com/Microsoft/Atlas/blob/master/src/Microsoft.Atlas.CommandLine/Templates/), but not sure whether to add a new file, or modify the existing code. What do you think?

Thanks!

**Comment by lodejard** [comment]
Hi @oneturkmen, yep that's the idea. This change shouldn't need any new classes - should be a change in this function:

https://github.com/Microsoft/Atlas/blob/master/src/Microsoft.Atlas.CommandLine/Commands/WorkflowCommands.cs#L287

The change for `operation.write` should be similar to the one that was made for `operation.message` --- it should work if you added a line right after this one:

https://github.com/Microsoft/Atlas/blob/master/src/Microsoft.Atlas.CommandLine/Commands/WorkflowCommands.cs#L298  

It would be almost identical to that line, but it would create a local variable for `operation.write` instead of `operation.message`. Then you should be able to change the rest of the references to `operation.write` in that function to use that new local variable instead.

**Comment by oneturkmen** [comment]
@lodejard, opened PR #56. Let me know if I did everything correctly. Thanks again for the help!

**Comment by lodejard** [comment]
Fixed by #56
